emcc.py

```diff
@@ -395,6 +395,34 @@ def apply_settings(user_settings):
     if key == 'WASM_OBJECT_FILES':
       settings.LTO = 0 if value else 'full'
 
+  wasm_bigint_min_browser_versions(user_settings)
+
+
+def wasm_bigint_min_browser_versions(user_settings):
```

Comment:
```suggestion
# apply minimum browser version defaults based on user settings. if
# a user requests a feature that we know is only supported in browsers
# from a specific version and above, we can assume that browser version.
def apply_min_browser_versions(user_settings):
```
I expect we may want to add this for more flags than just this one, so generalizing slightly.